triage-party/overlays/moc/smaug/secret-generator.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: viaduct.ai/v1
+kind: ksops
+metadata:
+  name: thoth-secret-generator
```

Comment:
Is this the thoth specific deployment?

Reply:
Nope, that's the op1st deployment - will fix the name, thanks.